### PR TITLE
Email user runbyuser

### PR DESCRIPTION
**Pull request by bentsi**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] ~~All new and existing unit tests passed (`hydra unit-tests`)~~
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: docker/env/hydra.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@ CENTOS_PREREQS_FILE=install-prereqs.sh
 WORK_DIR=/sct
 HOST_NAME=SCT-CONTAINER
 export SCT_TEST_ID=${SCT_TEST_ID:-$(uuidgen)}
+export SCT_GIT_USER_EMAIL=$(git config --get user.email)
```
Comment (fruch) [inline docker/env/hydra.sh]: using SCT_ would break sct_config since it's not documented option

Comment (bentsi) [reply docker/env/hydra.sh]: fixed

File: sdcm/remote.py [hunk 2]
```diff
@@ -148,7 +148,8 @@ def run(self, cmd, ignore_status=False, sudo=False, verbose=True, timeout=300):
                                            encoding='utf-8',
                                            hide=True,
                                            watchers=watchers,
-                                           command_timeout=timeout)
+                                           command_timeout=timeout,
```
Comment (fruch) [inline sdcm/remote.py]: would be good to test other code using local runner, to see this isn't breaking it

Comment (bentsi) [reply sdcm/remote.py]: I will run test-provision

Comment (bentsi) [reply sdcm/remote.py]: Test provision succeeded

**Pull request merged**